apps/bdd/features/create-wallet.feature

```diff
@@ -20,3 +20,18 @@ Feature: Create wallet
     When I fill in the wallet creation form with invalid data: wallet name: MyWallet
     And I click on the create wallet button
     Then I should see text "Wallet name already exists"
+
+
+  @discovery
```

Comment:
Tag `discovery` is a way to check the current sigutation, coderize it, reproduce bug, and so on. It should pass even there are error, catch the error in the code so people can reproduce it.